lnwire/neighbor_ack.go

```diff
@@ -10,18 +10,19 @@ import (
 )
 
 type NeighborAckMessage struct {
-	RoutingMessageBase
 }
 
 func (msg *NeighborAckMessage) String() string {
-	return fmt.Sprintf("NeighborAckMessage{%v %v}", msg.SenderID, msg.ReceiverID)
+	return fmt.Sprintf("NeighborAckMessage{}",)
 }
 
 func (msg *NeighborAckMessage) Command() uint32 {
 	return CmdNeighborAckMessage
 }
 
 func (msg *NeighborAckMessage) Encode(w io.Writer, pver uint32) error {
```

Comment:
In this case, a single byte can be written since the header on each message indicates the precise message type.